### DISCUSSION TITLE
Mirth User Group Unconference : 02 Feb 2022

### DISCUSSION THREAD

**Discussion by vinwi4** [nextgenhealthcare/connect#5613] [General]
https://nextdigitalhealth.com/mirthconnectunconference/ scheduled on 02 Feb from 6 PM EST

The Virtual Unconference for Mirth Connect offers a unique opportunity to learn from others and share your own experiences, while networking with professionals who can help you take things one step further. Plus - if ideas are flowing and coding skills at their peak – there's even an option of hacking away on plugins together! Don't miss this chance – become part of the integrated future now!

The Mirth Connect Unconference is the perfect place for you to learn more about this powerful healthcare integration platform. You’ll be able to share your own experiences and learn from others who are using Mirth Connect in innovative ways.​

Vineet
[https://www.wi4.org](https://wi4.org/)
https://nextdigitalhealth.com/

**Comment by JeffBenedict** [comment]
Looks like there's some html issues with the page you linked to?
![2023-01-19_012](https://user-images.githubusercontent.com/106398018/213549669-b4e7f944-42db-4729-97ab-33f739df0cc7.png)

**Reply by jonbartels** [reply]
OP is a spammer. This conference got spammed to MC GitHub a few weeks ago from a similarly named account and got banned.

Report the post and report the user.
